pull healthbar script into combat mechanics branch

sourceofentropy opened the pull request:
Need to pull healthbar script into combat mechanics branch and implement healthbar on current test model.

Apolleonysus committed:
first commit

wowowow

Apolleonysus committed:
had to remake my scene sorry

Apolleonysus committed:
combat scripts added

^

Apolleonysus committed:
figuring things out

lol

JaxsonVignal committed:
Merge pull request #12 from JaxsonVignal/GeoffreyP

Combat edit: HealthBar Script

Aleroth-byte committed:
Testingtesting

123

JaxsonVignal committed:
Merge pull request #13 from JaxsonVignal/PlayerController

Testingtesting

JaxsonVignal committed:
built basic movement and camera

JaxsonVignal commented:
you can access the scripts without merging the branches you just need to fetch the origin on gh desktop. the file should be available in any branch

JaxsonVignal committed:
added crouch and sprint

JaxsonVignal committed:
test

JaxsonVignal committed:
Update PlayerController_Test.unity

sourceofentropy commented:
@JaxsonVignal @Aleroth-byte

JaxsonVignal committed:
test

JaxsonVignal committed:
added slope handling

movement on slopes working now may need to adjust numbers

crazed6 committed:
Merge pull request #14 from JaxsonVignal/PlayerController

built basic movement and camera

JaxsonVignal committed:
added wallrun, slide, slope sliding, variable acceleration, and acceleration checks

still need to fix some issues with the wall run such as sticking while holding strafe. also need to add wall jump and adjust lerping gravity while on the wall. may also give the player control over y direction when on the wall (would need to remove lerping y on wall)

JaxsonVignal committed:
added wall jump

JaxsonVignal committed:
added coyote time and var gravity

character now has variable gravity when jumping and has coyote time implemented. Also restricted sliding with a short cd to prevent spamming for speed. still need to add jump cd to prevent double jump

JaxsonVignal committed:
tweaked some numbers

Aleroth-byte committed:
Merge pull request #18 from JaxsonVignal/PlayerController

added wallrun, slide, slope sliding, variable acceleration, and accel…

sourceofentropy commented:
Removing pull request. No longer necessary after @RyanSalam demonstrating correct process for pulling changes from main into your branch.
